### DISCUSSION TITLE
Asking about span parameters

### DISCUSSION THREAD

**Discussion by Palzar-Tamang** [Telecominfraproject/oopt-gnpy#489] [Q&A]
Hello there,
I did no understand what is meant by EOL in the span parameters?
Could anyone anyone help me with this ?

**Comment by EstherLerouzic** [comment]
Dear @Palzar-Tamang ,

EOL means end-of-life fiber repair margin that must be considered for the design of the network. Design will consider this margin as an additional loss for computing amplifiers'settings. For the time being it is added in the conn_out connector loss.

**Reply by Palzar-Tamang** [reply]
Thankyou @EstherLerouzic for the answer.
But does this mean that EOL value is kept zero, since it is being added to conn_out connector loss ?
